docs/en/guides/pipeline-integrations/development/sgtk-developer-app.md

```diff
@@ -0,0 +1,257 @@
+2. [Forking or downloading the starter app repository](#part-2---forking-or-downloading-the-starter-app-repository)
+3. [Adding the app to your config](#part-3---adding-the-app-to-your-config)
+4. [Developing the app](#part-4---developing-the-app)
+    - [Anatomy of the Template Starter App](#anatomy-of-the-template-starter-app)
+    - [Configuration settings](#configuration-settings)
+    - [Frameworks](#frameworks)
+    - [Reloading your changes](#reloading-your-changes)
+5. [Testing](#part-5---testing)
+6. [Preparing your first release](#part-6---preparing-your-first-release)
+
+Additional info:
+- [Modifying an existing app](#modifying-an-existing-app)
+    - [Pull requests](#pull-requests)
+
+## What is a Toolkit app?
+
+A Toolkit app can be defined as follows:
+
+- A Tool that is usually run by an end-user, using the Shotgun integrations.
+- They usually have a graphical user interface with which the user can guide the app's operations, but it doesn't have to, it can also be a command registered with the integrations, triggered by the user from the Shotgun menu in the software.
+- They can have an API/public methods in which other processes or apps could interact.
+- They can be multi-platform, and software agnostic.
+- It can be set up to be configured differently per [environment](https://developer.shotgunsoftware.com/487a9f2c/?title=Environment+Configuration+Reference#what-is-an-environment).
+- They can be contextually aware. In other words can know the task the user is currently working in and act accordingly.
+- They can only be run from a Toolkit engine.
+
+Toolkit apps are initialized by Toolkit engines. Engines are designed to run within a specific software environment, where they then provide an interface from which to run Toolkit apps. The engine abstracts away the complexity of needing to handle the various software environments from the app.
+This means the app only needs to focus on providing the functionality to fulfill its purpose and doesn't need to, for example, handle window parenting, keeping track of the user's context, or providing a short cut for launching itself.
+
+## Creating your own app.
+All apps and engines maintained and released by Shotgun Software are open source and you can find them in [GitHub](https://github.com/shotgunsoftware).
+
+This section goes through how to create a new app using our starter template. 
+We assume that you are familiar with GitHub and git workflows but please note that you can do Toolkit development even if you are not using git as your source control solution.
+
+
+## Part 1: Creating a development sandbox
+Before you do anything else, we recommend that you set up a [development sandbox by cloning the project configuration](../getting-started/installing_app.md#clone-the-pipeline-configuration-you-want-to-add-an-app-to).
+This will result in a separate configuration where you can develop your code, and test changes without affecting anyone else on the production. 
```

Comment:
- "...by cloning *your project's* configuration"? 
 - I'd omit the comma here: "develop your code, and test changes"